infrastructure/resources/web.tf

```diff
@@ -24,7 +24,8 @@ resource "azurerm_linux_web_app" "nbs_appts_web_app_service" {
   }
 
   app_settings = {
-    NBS_API_BASE_URL = "https://${azurerm_windows_function_app.nbs_appts_func_app.default_hostname}"
+    NBS_API_BASE_URL = "https://${azurerm_windows_function_app.nbs_appts_func_app.default_hostname}",
+    NBS_CLIENT_BASE_URL = "https://${azurerm_linux_web_app.nbs_appts_web_app_service.default_hostname}"
```

Comment:
As previously mentioned can we get client base from headers instead of relying on this?

Reply:
Done